mapactionpy_arcmap/MapCookbook.py

```diff
@@ -1,13 +1,14 @@
 from MapRecipe import MapRecipe
-import json 
+import json
+
 
 class MapCookbook:
     def __init__(self, cookbookJsonFile):
-        self.cookbookJsonFile=cookbookJsonFile
+        self.cookbookJsonFile = cookbookJsonFile
         self.products = list()
 
     def parse(self):
-        with open(self.cookbookJsonFile) as json_file:  
```

Comment:
As per the LayerProperties, I recommend calling `parse()` from within the constructor. I can't foresee a need to create a `MapCookbook` object (with the current constructor which requires a `cookbookJsonFile` parameter), without needing to parse the json file. Since that would mean that the only use of the `parse()` method would be to call it from within the constructor, then it could also be renamed `_parse()`.

Again same a LayerProperties, it would be helpful to test that all of the layers referenced within the MapCookbook, exist with a LayerProperties object (an by implication on disk). As far as I can tell it would not be a valid state of to have a MapCookbook object which references layers which are not properly described on with the LayerProperties object. Hence this test should be called by the constructor and it would be necessary to add a `lyrProperties` parameter to the constructor (or similar name). Some of this logic is currently in the `config_verify.py` file, though would be better if moved to the LayerProperties and MapCookbook.py files:
https://github.com/mapaction/mapactionpy_arcmap/pull/4/files#diff-01eac21ec2869a7f1f01805a2b1b0779